script/Arcueid/Day 13/13_00_ARC13_3.txt

```diff
@@ -172,38 +183,40 @@ My brain creaks.
 [sha:aa4a40757359c3088c41fec8cd6d4bd2172284a9]{
 -- Page 246, Offset 32348.
 -- 「……太陽じゃなくて、月の弱い明りだとよけいにはっきり視えるから。線をかき消すぐらい強い陽射しか、本当の暗闇のほうが、気が楽だ」
-"...Because unlike with the sun, the frail light of the moon makes everything clearly visible. I'd feel more comfortable with sunlight strong enough to erase the lines, or even with complete darkness."
+"...Because unlike the sun, the soft light of the moon reveals things I'd rather not see.
+ Sunlight strong enough to overpower the lines, or even complete darkness, would be more comfortable."
```

Comment:
on my first read and when nolo TLC'd it I didn't make the connection with day 4. I think this is a very good rendition.

that said, i think you should drop the last comma

Reply:
I didn't make that connection either, but sure I'll take credit for it